features/full_tests/native_feature_flags.feature

```diff
@@ -1,11 +1,10 @@
 Feature: Synchronizing feature flags to the native layer
 
   Scenario: Feature flags are synchronized to the native layer
-    When I run "CXXFeatureFlagNativeCrashScenario"
+    When I run "CXXFeatureFlagNativeCrashScenario" and relaunch the app
     And I configure Bugsnag for "CXXFeatureFlagNativeCrashScenario"
     And I wait to receive an error
     And the error payload contains a completed unhandled native report
-    And the exception "errorClass" equals "SIGILL"
```

Comment:
Does this change have some effect on error class...?

Reply:
The actual error name depends on Android version, so I've replaced this with:
```
And the exception "errorClass" equals one of:
          | SIGILL |
          | SIGTRAP |
```
Which is inline with the other similar scenarios